codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/ServerCodegenVisitor.kt

```diff
@@ -294,6 +294,7 @@ open class ServerCodegenVisitor(
                 this,
                 shape,
                 codegenDecorator.structureCustomizations(codegenContext, emptyList()),
+                structSettings = codegenContext.structSettings(),
```

Comment:
If there is no chance for the user to set this flag on the server side then this is okay. Otherwise, I think let's set this to:  `StructSettings(flattenVecAccessors = false)` for now.

Reply:
Disregard. Only the accessor is getting changed so let's keep it the way you have it right now.